interactions/api/gateway/client.py

```diff
@@ -390,7 +390,7 @@ def _dispatch_event(self, event: str, data: dict) -> None:
                 log.warning(
                     "Context is being created for the interaction, but no type is specified. Skipping..."
                 )
-        elif event != "TYPING_START":
+        elif event not in {"TYPING_START", "VOICE_STATE_UPDATE", "VOICE_SERVER_UPDATE"}:
```

Comment:
I think this breaks how the voice lib works, can you rather check in the error line?

Reply:
aytually nvm